code/datums/medal_awards.dm

```diff
@@ -78,3 +95,151 @@ GLOBAL_LIST_EMPTY(medal_awards)
 		return
 	if(give_medal_award(get_turf(printer)))
 		printer.visible_message(SPAN_NOTICE("[printer] prints a medal."))
+
+
+/proc/give_jelly_award(var/datum/hive_status/hive, var/attributed = TRUE)
```

Comment:
This entire file could use a once over for variable readability. Be explicit with the names of your variable so as to make it immediately identifiable as to what it is and what it's for.